Modix/Services/AutoCodePaste/CodePasteHandler.cs

```diff
@@ -26,16 +26,20 @@ public class CodePasteHandler
 */
 
 {4}";
-        internal static async Task MessageReceived(SocketMessage arg)
+        internal static string FormatHeader(IMessage arg)
         {
-            if (arg.Content.Length < 750) { return; }
-
-            string content = String.Format(Header, $"{arg.Author.Username}#{arg.Author.DiscriminatorValue}",
+            return String.Format(Header, $"{arg.Author.Username}#{arg.Author.DiscriminatorValue}",
                     arg.Channel.Name, DateTime.Now.ToString("dddd, MMMM d yyyy @ H:mm:ss"), arg.Id, FormatUtilities.FixIndentation(arg.Content));
+        }
+
+        internal static async Task MessageReceived(SocketMessage arg)
+        {
+            if (arg.Content.Length < 750 && arg.Content.Split('\n').Length <= 30) { return; } 
+            if (arg.Content.StartsWith("!exec") || arg.Content.StartsWith("!eval")) { return; } //let the eval module handle it
 
-            string url = await new CodePasteService().UploadCode(content);
+            string url = await new CodePasteService().UploadCode(FormatHeader(arg));
```

Comment:
Unrelated to this PR but why not a static method?

Reply:
It's also used in CodePasteService.cs itself; there's no reason *not* to have a header, though, so `UploadCode` should probably just call it for every input and I should keep the method private.